packages/marko-web-native-x/routes/story.js

```diff
@@ -0,0 +1,25 @@
+const { asyncRoute } = require('@parameter1/base-cms-utils');
```

Comment:
Rename the file to `with-story.js` once moved to the middleware folder.  When imported by the site, the usage should be this:

```js
const withNativeXStory = require('@parameter1/base-cms-marko-web-native-x/middleware/with-story.js');
const template = require('../templates/my-site-template');

app.use('/some-path', withNativeXStory({ template }));
```